### ISSUE TITLE
Update Omaha (Brave's auto-updater on Windows) to 1.3.36.131

### ISSUE THREAD

**Issue by mherrmann** [brave/brave-browser#23047]
## Description 

Google released a new upstream version of this technology. We should update Brave to it as well to get the latest fixes.

## Test plan

This change has the potential to break Brave's installers and entire update fleet on Windows. It is therefore very important to re-test these features once this issue is resolved. Please see https://github.com/brave/brave-core/pull/11096 for a test plan.

## Roll-out strategy

Please see https://github.com/brave/brave-core/pull/11096

**Comment by MadhaviSeelam** [comment]
Verification `In-progress` using 
```
Brave | 1.41.74 Chromium: 103.0.5060.53 (Official Build) beta (64-bit)
-- | --
Revision | a1711811edd74ff1cf2150f36ffa3b0dae40b17f-refs/branch-heads/5060@{#853}
OS | Windows 11 Version 21H2 (Build 22000.739)
```

**Comment by MadhaviSeelam** [comment]
@mherrmann `Updating the updater` issues: 

**Issue 1: Updater is not updating to correct version (`1.36.131`) with `Standalone silent installer`

1. Install standalone silent installer (1.39.96)
2. Have followed the test steps from `Case 1` https://github.com/brave/brave-browser/issues/23047#issuecomment-1163400663
3. Run `BraveSoftwareUpdateTaskMachineUA` from the Windows Task Scheduler I am seeing an error `The system cannot find the file specified. (0x80070002)'

Ex1|Ex2
---|---
<img width="962" alt="image" src="https://user-images.githubusercontent.com/98358127/176024942-5aadd2ab-f2a4-46a9-ab28-adce01a53647.png">|<img width="952" alt="image" src="https://user-images.githubusercontent.com/98358127/176025042-fa1c636c-db0d-4f79-8d00-ce71384166fa.png">

NOTE:  `Standalone installer` working as expected and I got the correct updater version `1.361.131` following Case 1 - https://github.com/brave/brave-browser/issues/23047#issuecomment-1163400663

**Issue 2: Update notifications: **
 `Brave` is not updating to latest version and not seeing `Brave Update` button 

1. Uninstall existing Omaha version
2. Install `1.39.96` standalone installer 
3. Confirmed build is installed successfully, and version details are shown correctly in brave://version
4. Close the browser
5. Configured my machine to pull updates from staging by setting the registry string value Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url to https://updates.brave.com/service/update2.
6. Killed all instances of BraveUpdate.exe, BraveCrashHandler.exe and BraveCrashHandler64.exe.
7. Delete the registry value HKLM\Software\Wow6432Node\BraveSoftware\Update\LastChecked - if it exists.
8. Run `BraveSoftwareUpdateTaskMachineUA` from the Windows Task Scheduler. Refresh with F5 until the task has ended.
9. Confirmed updater got updated to latest version which is 1.3.131 in the registry
10. Relaunch the browser and go to brave://settings/help
11. Brave didn't get updated latest version

ex1|ex2|ex3
---|---|---
<img width="951" alt="image" src="https://user-images.githubusercontent.com/98358127/176027974-2761932c-0729-47e7-b3ad-16bb671dc1b2.png">|<img width="951" alt="image" src="https://user-images.githubusercontent.com/98358127/176027988-1d23c90f-dea1-4a77-ae67-97a74e663b18.png">|<img width="951" alt="image" src="https://user-images.githubusercontent.com/98358127/176028024-d543b248-a919-49d2-a6c8-94c86d0098d6.png">

**Comment by mherrmann** [comment]
@MadhaviSeelam

Issue 1: Where did you see that error? I do see in the screenshot Ex1 that you have two copies of the updater installed: A copy just for your current Windows user and a copy for the entire machine. Which `Case 1` are you referring to? There are multiple `Case 1`s mentioned in the comment you linked to.

Issue 2: The latest Brave version on the staging server is 1.36.39. Once you've received the latest updater from the staging server, you could delete `UpdateDev\url` again to try to obtain the latest Brave from prod.

**Comment by MadhaviSeelam** [comment]
@mherrmann Here are more detailed steps for `Issue 1`.  The issue is I am unable to run the scheduler and omaha updates are not happening.

1. Uninstall all brave versions (all channels Dev, Release, Beta and Nightly) via Control panel
2. Uninstall existing Omaha version
3. Install standalone silent installer (1.39.96) 
4. Confirmed build is installed successfully, and version details are shown correctly in brave://settings/help
5. Confirmed that the old Omaha updater 1.3.361.113 is shown in the registry as expected
6. Go to brave://settings/help and wait for update and click on Relaunch and ensured brave is updated to the latest public beta version `1.41.82`
7. After upgrade, Omaha version still shows 1.3.361.113 as expected
8. Configure your machine to pull updates from staging by setting the registry string value `Computer\HKEY_CURRENT_USER\Software\BraveSoftware\UpdateDev\url` to `https://updates.brave.com/service/update2`
9. Kill all instances of `BraveUpdate.exe`, `BraveCrashHandler.exe` and `BraveCrashHandler64.exe`
10. Delete the registry value `Computer\HKEY_CURRENT_USER\Software\BraveSoftware\Update\LastChecked` - if it exists
11. Run `BraveSoftwareUpdateTaskMachineUA` from the Windows Task Scheduler.

Actual Results: When I try to run the scheduler, status is not switched to `Running`.  Eventually I see this error `The system cannot find the file specified` in the line item for `BraveSoftwareUpdateTaskMachineUA`

Ex1|Ex2|Ex3
---|---|----
<image width="860" alt="image" src="https://user-images.githubusercontent.com/98358127/176301935-ac05f076-f95e-4f9a-aa1b-b590ac732517.png">|<image width="860" alt="image" src="https://user-images.githubusercontent.com/98358127/176301887-67af9813-1c8b-4d3f-bb39-a79d5799e5e3.png">|<image width="860" alt="image" src="https://user-images.githubusercontent.com/98358127/176301978-0a61d0d9-42ff-4596-9561-0099a80aec0e.png">

**Comment by mherrmann** [comment]
@MadhaviSeelam Issue 2: I think the `Brave Update` button only appears after some time. I wouldn't expect it to show up immediately.

Issue 1:

Omaha can be installed just for the current user, or for the entire machine. If it's installed for the current user only, then you get `BraveSoftwareUpdateTaskUser...` in the Task Scheduler, registry entries in `HKEY_CURRENT_USER\...` and Omaha's files in `%LOCALAPPDATA%`. On the other hand, if it's installed for the entire machine, then you get `BraveSoftwareUpdateTaskMachine...` in the Task Scheduler, registry entries in `HKEY_LOCAL_MACHINE\...` and Omaha's files in `C:\Program Files (x86)`.

The steps you posted mention `HKEY_CURRENT_USER` - which would be for a per-user install of Omaha - but also `BraveSoftwareUpdateTaskMachineUA` - which is for a machine-wide install. My feeling is that the discrepancies you are experiencing come from the mix of the two. Furthermore, I believe the specific value `...\UpdateDev\url` always needs to be set in `HKEY_LOCAL_MACHINE`.

I believe I would try to remove both the per-user and the per-machine install of Omaha from your system, then repeat any steps with either a machine- or user install. To make sure that all copies of Omaha were successfully uninstalled, you can check that there are no `BraveSoftwareUpdate` tasks of any kind in the Task Scheduler, there is no `Update\Clients` key for BraveSoftware in either `HKEY_CURRENT_USER` or `HKEY_LOCAL_MACHINE`, and no `BraveUpdate.exe` in either `%LOCALAPPDATA%` and `C:\Program Files (x86)`.

**Comment by GeetaSarvadnya** [comment]
Verification is in-progress

```
Brave | 1.41.82 Chromium: 103.0.5060.66 (Official Build) beta (64-bit)
-- | --
Revision | 20b1569438a85e631d15e83eb355e3e326e5da6f-refs/branch-heads/5060@{#1066}
OS | Windows 10 Version 21H2 (Build 19044.1766)

```
- Verified the test plan from https://github.com/brave/brave-core/pull/11096

**Comment by LaurenWags** [comment]
Verification on Windows 10 x86 is in progress with

```

```

- Verified the same test cases used to verify last Omaha bump from https://github.com/brave/brave-browser/issues/22060#issuecomment-1130185884

<details>
<summary>Pre-requisite steps (largely not used below as I use a fresh VM instead of cleaning up)</summary>

**Pre-requisites:**
(Please note these steps are same as remove updater as described in the "Cleaning up after a test" in https://github.com/brave/brave-core/pull/11096 but I have breakdown the steps into detailed)

1. Uninstall all the brave version (all channels Dev, Release, Beta and Nightly) via Control panel
2. Remove all the local profile folders from `%LOCALAPPDATA%\BraveSoftware`
3. Go to `C:\Program Files (x86)\BraveSoftware\update` path and observe that `Update` folder is still available and updater version `1.3.361.113` is still shown
4. Open RegEdit and navigate to `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update` and ensure Omaha updater version is `1.3.361.113` 
5. Open command prompt with Admin privileges and run the command `"C:\Program Files (x86)\BraveSoftware\Update\BraveUpdate.exe" /uninstall`
6. Go to C:\Program Files (x86)\BraveSoftware path and observe that the `update` folder is removed
7. Open RegEdit and navigate to `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update` and ensure Omaha updater version `1.3.361.113` is removed from the registry
8. If the registry key isn't empty in steps 6 and 7 then kill all instances of `BraveUpdate.exe`, `BraveCrashHandler.exe`, `BraveCrashHandler64.exe` in the Task Manager
9. Steps 6 and 7 indicate that the Omaha updater is completely uninstalled

</details>

### Windows 10 x86
**_1. Verify installers and auto-updater work on a clean profile_**

<details>
<summary>Case 1: Verify the Omaha updater version when 1.41.x is installed - InProgress</summary>

**Steps:**
1. Run the steps 1 to 7 from Pre-requisites above - N/A as I used a fresh VM with Brave never having been installed
2. Download `BraveBrowserStandaloneBetaSetup32.exe` for 1.39.106 and install the standalone binary 1.39.x with Admin rights and ensured that standalone installer installed successfully
3. Confirmed that the new Omaha updater `1.3.361.113` is shown in the registry as expected
4. Confirmed new Omaha updater version `1.3.361.113` is shown in `C:\Program Files (x86)\BraveSoftware\Update`
5. Uninstall the 1.39.x from control panel and remove the profile folder from the `%LOCALAPPDATA%\BraveSoftware`
6. Install the 1.39.x again (installed 1.38.101 this time) and confirmed that the brave installed successfully
7. Go to brave://settings/help and wait for the update and and click on the `Relaunch` and confirmed that brave is upgraded to latest public version

Step 3 | Step 4 | Step 6 | Step 7
-- | -- | -- | --

 
</details>

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 8.1 x64` using the following build(s):

```
Brave | 1.41.95 Chromium: 103.0.5060.114 (Official Build) (64-bit)
--- | ---
Revision | a1c2360c5b02a6d4d6ab33796ad8a268a6128226-refs/branch-heads/5060@{#1124}
OS | Windows 8.1 (Build 9600.20402)
```

- Verified the same test cases used to verify last Omaha bump from https://github.com/brave/brave-browser/issues/22060#issuecomment-1130185884

<details>
<summary>Pre-requisite steps (largely not used below as I use a fresh VM instead of cleaning up)</summary>

**Pre-requisites:**
(Please note these steps are same as remove updater as described in the "Cleaning up after a test" in https://github.com/brave/brave-core/pull/11096 but I have breakdown the steps into detailed)

1. Uninstall all the brave version (all channels Dev, Release, Beta and Nightly) via Control panel
2. Remove all the local profile folders from `%LOCALAPPDATA%\BraveSoftware`
3. Go to `C:\Program Files (x86)\BraveSoftware\update` path and observe that `Update` folder is still available and updater version `1.3.361.113` is still shown
4. Open RegEdit and navigate to `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update` and ensure Omaha updater version is `1.3.361.113` 
5. Open command prompt with Admin privileges and run the command `"C:\Program Files (x86)\BraveSoftware\Update\BraveUpdate.exe" /uninstall`
6. Go to C:\Program Files (x86)\BraveSoftware path and observe that the `update` folder is removed
7. Open RegEdit and navigate to `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update` and ensure Omaha updater version `1.3.361.113` is removed from the registry
8. If the registry key isn't empty in steps 6 and 7 then kill all instances of `BraveUpdate.exe`, `BraveCrashHandler.exe`, `BraveCrashHandler64.exe` in the Task Manager
9. Steps 6 and 7 indicate that the Omaha updater is completely uninstalled

</details>

### Windows 8.1 x64
**_1. Verify installers and auto-updater work on a clean profile_**

<details>
<summary>Case 1: Verify the Omaha updater version when 1.41.x is installed - PASSED</summary>

**Steps:**

1. Run the steps 1 to 7 from Pre-requisites above - N/A as I used a fresh VM with Brave never having been installed
2. Download `BraveBrowserStandaloneBetaSetup.exe` for `1.41.95` and install the standalone binary `1.41.x` with Admin rights and ensured that standalone installer installed successfully
3. Confirmed that the new Omaha updater `1.3.361.131` is shown in the registry as expected
4. Confirmed new Omaha updater version `1.3.361.131` is shown in `C:\Program Files (x86)\BraveSoftware\Update`
5. Uninstall the 1.41.x from control panel and remove the profile folder from the `%LOCALAPPDATA%\BraveSoftware`
6. Install the 1.41.x again (installed `1.41.86` this time) and confirmed that the brave installed successfully
7. Go to brave://settings/help and wait for the update and and click on the `Relaunch` and confirmed that brave is upgraded to latest public version

Step 3 | Step 4 | Step 6 | Step 7
-- | -- | -- | --
<img width="1285" alt="Screen Shot 2022-07-11 at 11 22 22 PM" src="https://user-images.githubusercontent.com/2602313/178402222-83670ac2-2cf4-411a-bef5-f0d2a54ae5c5.png"> | <img width="820" alt="Screen Shot 2022-07-11 at 11 22 40 PM" src="https://user-images.githubusercontent.com/2602313/178402244-7ac202fe-23df-4d81-9e13-5280c9039561.png"> | <img width="1249" alt="Screen Shot 2022-07-11 at 11 25 03 PM" src="https://user-images.githubusercontent.com/2602313/178402264-e5ec2019-6fae-46d0-aa1c-449dc336214b.png"> | <img width="1222" alt="Screen Shot 2022-07-11 at 11 25 50 PM" src="https://user-images.githubusercontent.com/2602313/178402277-15e28664-b970-434d-bd60-542266b8fc90.png">

</details>

**_2. Verify installers and auto-updater works on upgrade profile_**

<details>
<summary>Case 1:Verify the Omaha updater version in an upgraded profile (Install 1.37.77 ->upgrade to 1.41.x) - PASSED</summary>

Steps :

1. Follow the pre-requisites steps from case 1 to uninstall the Omaha - N/A as I used a fresh VM without Brave ever being installed
2. Install the `BraveBrowserStandaloneBetaSetup.exe` build version `1.37.77` (build before the Omaha updater change `1.3.361.131`)
3. Confirmed build is installed successfully and version details are shown correctly in brave://settings/help
4. Confirmed that the old Omaha updater `1.3.361.111` is shown in the registry as expected
5. Confirmed old Omaha updater version `1.3.361.111` is shown in `C:\Program Files (x86)\BraveSoftware\Update`
6. Go to brave://settings/help and wait for update and click on `Relaunch` and ensured brave is updated to the latest public beta version 1.41.x

`1.37.77`

Step 4 | Step 5 | Step 6 | Step 6
-- | -- | -- | --
<img width="1288" alt="Screen Shot 2022-07-11 at 11 32 37 PM" src="https://user-images.githubusercontent.com/2602313/178403127-3e985c4a-cfb3-4e6f-b574-870f070239c4.png"> | <img width="815" alt="Screen Shot 2022-07-11 at 11 32 54 PM" src="https://user-images.githubusercontent.com/2602313/178403128-ab7caba9-2bf2-46c7-8db7-8cc08537e9cf.png"> | <img width="1444" alt="Screen Shot 2022-07-11 at 11 33 31 PM" src="https://user-images.githubusercontent.com/2602313/178403129-4e123171-fc1f-429d-948f-f593dfff8cff.png"> | <img width="1441" alt="Screen Shot 2022-07-11 at 11 33 44 PM" src="https://user-images.githubusercontent.com/2602313/178403130-137e72f9-7fc7-439b-84fa-9e348561d85d.png">

Used the STR/Cases outlined via 1.) https://github.com/brave/brave-browser/issues/22060#issuecomment-1124861678 and then 2.) "Background updates" in https://github.com/brave/brave-core/pull/11096 - to verify the Omaha updates silently in the background and confirmed that updater got updated to latest version `1.3.361.131`.

Confirmed Omaha updater version updated from `1.3.361.111` --> `1.3.361.131`

Example | Example
----- | -----
<img width="1295" alt="Screen Shot 2022-07-12 at 12 06 23 AM" src="https://user-images.githubusercontent.com/2602313/178406536-d444fc52-0b46-4593-b38c-6a824cca5742.png"> | <img width="815" alt="Screen Shot 2022-07-12 at 12 06 08 AM" src="https://user-images.githubusercontent.com/2602313/178406551-dc1f5655-1098-46b2-af82-e87160a6055c.png">

</details>

<details>
<summary>Case 2: Download and install stub installer binary 1.41.x with Admin rights - PASSED</summary>

- Uninstall 1.41.x from the control panel if it's installed - N/A as I'm using a fresh VM and Brave has never been installed
- Remove local profile folders from `%LOCALAPPDATA%\BraveSoftware` - N/A as I'm using a fresh VM and Brave has never been installed
- Install 1.41.x  stub installer `BraveBrowserSetup-64-r-test.exe` and ensured that Brave is installed successfully and latest public released beta version is fetched from the Omaha server as expected
- Install 1.40.x stub installer `BraveBrowserSetup.exe` from brave.com and ensured that Brave is installed successfully and the latest public released versions is fetched from the Omaha server as expected.

```
Brave | 1.40.113 Chromium: 103.0.5060.114 (Official Build) (64-bit)
--- | ---
Revision | a1c2360c5b02a6d4d6ab33796ad8a268a6128226-refs/branch-heads/5060@{#1124}
OS | Windows 8.1 (Build 9600.20402)
```

```
Brave | 1.41.95 Chromium: 103.0.5060.114 (Official Build) (64-bit)
--- | ---
Revision | a1c2360c5b02a6d4d6ab33796ad8a268a6128226-refs/branch-heads/5060@{#1124}
OS | Windows 8.1 (Build 9600.20402)
```

</details>

<details>
<summary>Case 3: Download and install standalone silent installer binary 1.41.x - PASSED</summary>

- Uninstall 1.41.x from the control panel if it's installed - N/A (fresh VM)
- Remove local profile folders from `%LOCALAPPDATA%\BraveSoftware` - N/A (fresh VM)
- Install 1.41.x  standalone silent installer `BraveBrowserStandaloneSilentBetaSetup.exe` and ensured that Brave is installed in silent mode (Installer modal is NOT shown on the UI) and ensured that correct version details are shown `brave://settings/help`

```
Brave | 1.41.95 Chromium: 103.0.5060.114 (Official Build) (64-bit)
--- | ---
Revision | a1c2360c5b02a6d4d6ab33796ad8a268a6128226-refs/branch-heads/5060@{#1124}
OS | Windows 8.1 (Build 9600.20402)
```

</details>

**_3. Verify installers and auto-updater work when the user declines the Admin rights in UAC_**

<details>
<summary>Case 1: Verify standalone installer can be installed without admin rights and updater works - PASSED</summary>

- Install 1.41.x beta standalone installer
- Decline the Admin rights in UAC while installing the setup
- Confirmed 1.41.x beta standalone installer is installed successfully
- Go to brave://settings/help and wait for the update and click on `Relaunch` 
- Confirmed that the brave is upgraded to 1.41.x beta public released version

Installed `1.41.86` --> Updated to `1.41.91`

Example | Example | Example | Example | Example
----- | ----- | ----- | ----- | -----
<img width="455" alt="Screen Shot 2022-07-12 at 12 22 42 AM" src="https://user-images.githubusercontent.com/2602313/178409860-f1531feb-04c1-44d4-a5fe-49cddc0d07fd.png"> | <img width="502" alt="Screen Shot 2022-07-12 at 12 22 32 AM" src="https://user-images.githubusercontent.com/2602313/178409859-d93ca88d-fa3f-4751-8cbd-2d545343fb00.png"> | <img width="1300" alt="Screen Shot 2022-07-12 at 12 23 44 AM" src="https://user-images.githubusercontent.com/2602313/178409858-7821ad73-60e4-418f-8066-f9a18ef285e9.png"> | <img width="1380" alt="Screen Shot 2022-07-12 at 12 24 26 AM" src="https://user-images.githubusercontent.com/2602313/178409856-14f59190-1efb-41e4-abe2-f573c4ac661c.png"> | <img width="1400" alt="Screen Shot 2022-07-12 at 12 24 36 AM" src="https://user-images.githubusercontent.com/2602313/178409852-c4e178d3-11c5-4a45-a712-57e35a209e53.png">

</details>

**_4. Verify re-install same installer version works_**

<details>
<summary>Case 1: Verify standalone installer can be installed twice and update works - PASSED</summary>

- Install 1.41.x standalone installer twice 
- Ensured two brave windows are opened with the same version
- Go to brave://settings/help and wait for the update and click on `Relaunch`
- Confirmed that brave is updated to 1.41.x public released version

Example | Example
----- | -----
<img width="1708" alt="Screen Shot 2022-07-12 at 12 59 02 AM" src="https://user-images.githubusercontent.com/2602313/178412221-8f4e39d3-2946-4045-a8a0-30f7f30ff35d.png"> | <img width="1721" alt="Screen Shot 2022-07-12 at 12 59 31 AM" src="https://user-images.githubusercontent.com/2602313/178412230-cb65c826-1118-4aff-953e-90fea6741a79.png">

</details>

**_5. Install quite older installers and check update works correctly_**

<details>
<summary>Case 1: Install 1.30.x and ensure upgrade works - PASSED</summary>

- Install 1.30.x beta build
- Go to brave://settings/help and wait for update and click on `Relaunch` 
- Confirmed 1.41.x public released version is installed

1.30.x | 1.41.x
-- | --
<img width="1343" alt="Screen Shot 2022-07-12 at 1 04 20 AM" src="https://user-images.githubusercontent.com/2602313/178412801-8cc54ee2-c12d-4722-b411-ad288df4a0fc.png"> | <img width="1356" alt="Screen Shot 2022-07-12 at 1 04 30 AM" src="https://user-images.githubusercontent.com/2602313/178412808-9382013e-708a-4dd6-b7bb-0f4c2b3922c3.png">

</details>

**_6. Verify that install brave works when the Internet is disconnected_**

<details>
<summary>Case 1: Verify standalone installer can be installed when the network is disconnected - PASSED</summary>

- Disconnect network and Install 1.41.86 standalone installer
- Confirmed standalone installer is installed successfully and version details are shown correctly
- Go to brave://settings/help and ensured update is NOT shown which is expected as to pull the Omaha version network needs to be connected
- Reconnect the network
- Confirm upgrade is fetched
- Confirm successful upgrade

Network disconnected | Network disconnected | Reconnect Network | Upgrade successful
-- | -- | -- | --
<img width="1714" alt="Screen Shot 2022-07-12 at 1 10 26 AM" src="https://user-images.githubusercontent.com/2602313/178413600-ce95ef94-148b-4a46-b86f-7cd54a2d5c7c.png"> | <img width="1706" alt="Screen Shot 2022-07-12 at 1 10 36 AM" src="https://user-images.githubusercontent.com/2602313/178413604-3bbb2328-2628-46bb-8f47-71e37df98ce1.png"> | <img width="1069" alt="Screen Shot 2022-07-12 at 1 11 29 AM" src="https://user-images.githubusercontent.com/2602313/178413611-1b6d826e-6167-4e3d-81db-99254d7b4094.png"> | <img width="1065" alt="Screen Shot 2022-07-12 at 1 11 37 AM" src="https://user-images.githubusercontent.com/2602313/178413616-674b3572-bad5-46e1-a710-eed5d5568cd8.png">

</details>

**_7. Verify installing a new binary by older binary from different installers (e.g one is standalone and the other is stub/silent)_**

<details>
<summary>Case 1: Install standalone installer 1.40.x and upgrade it by 1.41.x stub installer - PASSED</summary>

- Disabled network
- Clean profile 1.40.x standalone installer `BraveBrowserStandaloneSetup.exe`
- Install the standalone installer and confirmed 1.40.x is installed successfully
- Close 1.40.x
- Stop/remove all upgrade processes that are running via Task Manager
- Reenable network
- Upgrade the profile by 1.41.x stub installer `BraveBrowserBetaSetup32.exe`
- Confirmed that the 1.41.x public releases version is installed successfully

1.40.x | 1.40.x | 1.41.x | 1.41.x
----- | ----- | ----- | -----
<img width="1104" alt="Screen Shot 2022-07-12 at 1 19 10 AM" src="https://user-images.githubusercontent.com/2602313/178414802-1c245925-2585-458d-83f0-1f45d579c7c8.png"> | <img width="1107" alt="Screen Shot 2022-07-12 at 1 19 42 AM" src="https://user-images.githubusercontent.com/2602313/178414813-8db6dcbc-7a5a-491d-8a9d-bdbe03579856.png"> | <img width="1088" alt="Screen Shot 2022-07-12 at 1 22 31 AM" src="https://user-images.githubusercontent.com/2602313/178414844-cd6f1301-fe02-41d6-b474-255fc99570e2.png"> | <img width="1095" alt="Screen Shot 2022-07-12 at 1 22 37 AM" src="https://user-images.githubusercontent.com/2602313/178414850-4448f1a9-9c66-49ae-876b-ee6ea823ff4a.png">

</details>
